README.adoc

```diff
@@ -271,20 +375,61 @@ system-deployment-694c7b74f7-lrlf7     0/1       Running   0          1m
 inventory-deployment-cf8f564c6-nctcr   1/1       Running   0          1m
 ----
 
-You will notice that one of the two `system` pods is no longer in the ready state. Navigate 
-to `{system-api}/properties`. Observe that your request will still be successful because you have two 
-replicas and one is still healthy.
+// Static guide instruction
+ifndef::cloud-hosted[]
+You will notice that one of the two `system` pods is no longer in the ready state.
+Navigate to `{system-api}/properties`.
+Observe that your request will still be successful because you have two replicas and one is still healthy.
+endif::[]
+
+// Cloud hosted guide instruction
+ifdef::cloud-hosted[]
+You will notice that one of the two **system** pods is no longer in the ready state.
+Make a request to the **/system/properties** endpoint with the following command:
+```
+curl -s http://$SYSTEM_PROXY/system/properties | jq
+```
+{: codeblock}
+
+Observe that your request will still be successful because you have two replicas and one is still healthy.
+endif::[]
 
 === Observing the effects on the inventory microservice
 
-Wait until the `system` pod is ready again. Make two POST requests to `{system-api}/unhealthy`.
- If you see the same pod name twice, make the request again until you see that the second 
- pod has been made unhealthy. You may see the same pod twice because there's a delay 
- between a pod becoming unhealthy and the readiness probe noticing it. Therefore, traffic 
- may still be routed to the unhealthy service for approximately 5 seconds. Continue to 
- observe the output of `kubectl get pods`. You will see both pods are no longer ready. 
- During this process, the readiness probe for the `inventory` microservice will also fail. 
- Observe it's no longer in the ready state either.
+// Static guide instruction
+ifndef::cloud-hosted[]
+Wait until the `system` pod is ready again.
+Make two requests to `{system-api}/unhealthy`.
+If you see the same pod name twice, make the request again until you see that the second 
+pod has been made unhealthy. You may see the same pod twice because there's a delay 
```

Comment:
You may see the same pod twice because there's a delay between a pod becoming unhealthy and the readiness probe noticing it.

---->

You might see the same pod twice due to a delay between when a pod becomes unhealthy and when the readiness probe notices it.